Automattic/newspack-popups
# perf: shorten query params for preview iframe URLs

### All Submissions:






### Changes proposed in this Pull Request:



Shortens the preview URLs used to render prompts in a preview iframe. This should help avoid a bug in Chrome where iframes with very long URLs sometimes refuse to resolve.

### How to test the changes in this Pull Request:

1. On a server-based test site (not a local environment), using Chrome, preview a single prompt from the editor screen. You may have to repeat several times, but eventually the preview modal will hang at the "Loading..." view.
2. Check out this branch, rebuild.
3. In a new Chrome session, attempt to replicate the bug in step 1 again. You shouldn't be able to, theoretically.

### Other information:
